db/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mariadb:10.2
+
+ENV MYSQL_USER=beaker \
+      MYSQL_PASSWORD=beaker \
+      MYSQL_ALLOW_EMPTY_PASSWORD=yes
+
+COPY ./my.cnf /etc/mysql/my.cnf
+RUN chown root.root /etc/mysql/my.cnf && chmod 0644 /etc/mysql/my.cnf
```

Comment:
What about SE?

Reply:
I did not worry about it because those commands come from Ansible code, and they do exactly what was there. And I got no errors: maybe the container has a permissive state?